_posts/atoms/2018-09-27-typography.html

```diff
@@ -2,41 +2,12 @@
 layout: post
 title: Typography
 category: atoms
-description: We stick to [Bootstrap's global settings for typography](https://getbootstrap.com/docs/4.1/content/typography/#global-settings).
-             We use Bootstrap [typography](https://getbootstrap.com/docs/4.1/content/typography/) and [text](https://getbootstrap.com/docs/4.1/utilities/text/) classes when needed.
+description: After some research we decided to stick with system fonts as well known as [Native font stack](https://getbootstrap.com/docs/4.1/content/reboot/#native-font-stack). 
              We use the [default fonts from Bootstrap](https://getbootstrap.com/docs/4.1/content/reboot/#native-font-stack) with the default font-size of `16px`.
+             We use Bootstrap [typography](https://getbootstrap.com/docs/4.1/content/typography/) and [text](https://getbootstrap.com/docs/4.1/utilities/text/) classes when needed.
 ---
 
-%hr
-%h5 Headings
-%hr
-
-%h1 h1 Heading 1
-%h2 h2 Heading 2
-%h3 h3 Heading 3
-%h4 h4 Heading 4
-%h5 h5 Heading 5
-%h6 h6 Heading 6
-
-%p.h1 Paragraph with .h1
-%p.h2 Paragraph with .h2
-%p.h3 Paragraph with .h3
-%p.h4 Paragraph with .h4
-%p.h5 Paragraph with .h5
-%p.h6 Paragraph with .h6
-
-%h1
-  h1 Heading 1
-  %small.text-muted with faded text
 
-%hr
-%h5 Inline Text Elements
```

Comment:
We should keep this as there are still other inline text elements even though `%mark` is now removed.